tests/auth_tests/test_forms.py

```diff
@@ -867,6 +869,31 @@ def test_bug_19349_bound_password_field(self):
         # value to render correctly
         self.assertEqual(form.initial["password"], form["password"].value())
 
+    @override_settings(ROOT_URLCONF="auth_tests.urls_admin")
+    def test_bug_34066_link_to_password_reset_in_helptext(self):
+        user = User.objects.get(username="testclient")
+        form = UserChangeForm(data={}, instance=user)
+        passsword_help_text = form.fields["password"].help_text
+        matches = re.search('<a href="(.*?)">', passsword_help_text)
+
+        # Get URL to UserChangeForm in admin via to_field (instead of pk)
+        admin_user_change_url = f"""{reverse(
+            "admin:%s_%s_change" % (user._meta.app_label, user._meta.model_name),
+            args=(user.username,),
+        )}?to_field=username"""
+
+        # join relative url found in password's help_text to admin_user_change_url
+        joined_url = urllib.parse.urljoin(admin_user_change_url, matches.group(1))
+
+        # Get URL to PasswordChangeForm in admin
+        pw_change_url = reverse(
+            "admin:%s_%s_password_change"
+            % (user._meta.app_label, user._meta.model_name),
+            args=(user.pk,),
+        )
+        # assert joined_url and pw_change_url are ideential
```

Comment:
Just got a typo here 😁
```suggestion
        # assert joined_url and pw_change_url are identical
```

Reply:
Actually the comment is rather superfluous… it's usually better the comment the why and not the what
```suggestion
```

Reply:
To add more explanation, comments should state the expected behavior and omit prefixes like "Assert" since all tests assert things. This guideline is from [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).